### ISSUE TITLE
Warning from relion_display after 2D class selection

### ISSUE THREAD

**Issue by psryubh** [3dem/relion#275]
Hello!

I am using relion2.1beta1 which has been available last night.

And I am stuck in some minor problem.

When I run subset selection to sort the good classes from the 2D classification results,
somehow everything seems to work, but I've got warning message; "you cannot sort particles and use a maximum number of images. Ignoring the latter...".

Apparently, the number of the selected particles decrease from the total number of the auto-picked particles (from ~1000 to 844 particles). I don't know what's wrong. Should I concern this warning? 
In fact, I have also bumped with this warning when I used relion2.1.beta0 as well.

The followings are run.out and run.err file contents.

**The following is note.txt**
 ++++ Executing new job on Fri Aug 18 14:44:47 2017
 ++++ with the following command(s): 
`which relion_refine_mpi` --o Class2D/job031/run --i ./Extract/job025/particles.star --dont_combine_weights_via_disc --pool 3 --ctf  --iter 25 --tau2_fudge 2 --particle_diameter 190 --K 50 --flatten_solvent  --zero_mask  --oversampling 1 --psi_step 10 --offset_range 5 --offset_step 2 --norm --scale  --j 1 --gpu "0:1:0:1" 
 ++++ 

**The following is run.out**
Reading in all images...
   0/   0 sec ............................................................~~(,_,">
Reading in all images...
   1/   1 sec ............................................................~~(,_,">
 Written out Select/job032/backup_selection.star
Saved Select/job032/class_averages.star with 11 selected images.
Saved Select/job032/particles.star with 844 selected particles.

**The following is run.err**
 WARNING: you cannot sort particles and use a maximum number of images. Ignoring the latter...
 WARNING: you cannot sort particles and use a maximum number of images. Ignoring the latter...

**Comment by bforsbe** [comment]
A subset selection is exactly that; less than all input particles. Why would you expect all of the particles to be selected?

Regarding the warning, in the bottom right of the display window that pops up as you hit "run" you can set how many images to show. Just set that to -1 and the warning should go away.

**Comment by psryubh** [comment]
Thanks for the comments, Bjorn!
Setting the Max. nr. images to -1 blew out the warning. It was very simple thing..(I've just started using Relion)

When it comes to the selection thing,
I did not expect to see all of particles to be selected since I manually selected less than all. Rather, I just mentioned it as a proof that my selection was valid against the warning. 
Now I understood that the warning is totally not related to the selection itself.

Again, thanks to your comments for my too much beginner-question.
